internal/distro/rhel8/pipelines.go

```diff
@@ -670,6 +670,16 @@ func osPipeline(t *imageType,
 	}
 
 	if t.rpmOstree {
+		p.AddStage(osbuild.NewSystemdJournaldStage(
```

Comment:
@achilleas-k Seems like a straight forward change to me, is there any change with the manifests regeneration `go run ./cmd/gen-manifests -output test/data/manifests -workers 50 -images edge-commit,edge-container`, the tests are failing.